more python 3.5 failures

```python
+ nosetests-3.5 kitchen3/tests/
........E...E.............F..E..FE.......................................S................S................................E...E.....................................................................................................................................................................................................................................................................................................................................................................................................................................F..F................S....................
======================================================================
ERROR: For each module, check that every public name is in __all__
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test__all__.py", line 163, in check__all__is_complete
    exec('from %s import %s' % (modpath, modname), names)
  File "<string>", line 1, in <module>
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/kitchen/pycompat24/subprocess.py", line 4, in <module>
    import kitchen.pycompat27.subprocess as __s
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/kitchen/pycompat27/subprocess/__init__.py", line 46, in <module>
    from subprocess import list2cmdline, mswindows
ImportError: cannot import name 'mswindows'
======================================================================
ERROR: For each module, check that every public name is in __all__
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test__all__.py", line 163, in check__all__is_complete
    exec('from %s import %s' % (modpath, modname), names)
  File "<string>", line 1, in <module>
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/kitchen/pycompat27/subprocess/__init__.py", line 46, in <module>
    from subprocess import list2cmdline, mswindows
ImportError: cannot import name 'mswindows'
======================================================================
ERROR: For each name in module's __all__, check that it exists
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test__all__.py", line 124, in check_everything_in__all__exists
    exec('from %s import %s' % (modpath, modname), names)
  File "<string>", line 1, in <module>
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/kitchen/pycompat24/subprocess.py", line 4, in <module>
    import kitchen.pycompat27.subprocess as __s
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/kitchen/pycompat27/subprocess/__init__.py", line 46, in <module>
    from subprocess import list2cmdline, mswindows
ImportError: cannot import name 'mswindows'
======================================================================
ERROR: For each name in module's __all__, check that it exists
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test__all__.py", line 124, in check_everything_in__all__exists
    exec('from %s import %s' % (modpath, modname), names)
  File "<string>", line 1, in <module>
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/kitchen/pycompat27/subprocess/__init__.py", line 46, in <module>
    from subprocess import list2cmdline, mswindows
ImportError: cannot import name 'mswindows'
======================================================================
ERROR: test_deprecation_py3.TestPendingDeprecationModules.test_modules('subprocess', 'kitchen.pycompat24.subprocess', 'kitchen/pycompat24')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test_deprecation_py3.py", line 42, in check_modules
    importlib.find_loader(module_fqn, module_path).load_module()
AttributeError: 'NoneType' object has no attribute 'load_module'
======================================================================
ERROR: test_deprecation_py3.TestPendingDeprecationModules.test_modules('subprocess', 'kitchen.pycompat27.subprocess', 'kitchen/pycompat27')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test_deprecation_py3.py", line 42, in check_modules
    importlib.find_loader(module_fqn, module_path).load_module()
AttributeError: 'NoneType' object has no attribute 'load_module'
======================================================================
FAIL: For each name in module's __all__, check that it exists
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test__all__.py", line 131, in check_everything_in__all__exists
    exec('from %s.%s import *' % (modpath, modname), interior_names)
  File "<string>", line 1, in <module>
ImportError: cannot import name 'mswindows'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test__all__.py", line 135, in check_everything_in__all__exists
    modname, e.__class__.__name__, e))
AssertionError: __all__ failure in pycompat24: ImportError: cannot import name 'mswindows'
======================================================================
FAIL: For each name in module's __all__, check that it exists
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test__all__.py", line 131, in check_everything_in__all__exists
    exec('from %s.%s import *' % (modpath, modname), interior_names)
  File "<string>", line 1, in <module>
ImportError: cannot import name 'mswindows'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test__all__.py", line 135, in check_everything_in__all__exists
    modname, e.__class__.__name__, e))
AssertionError: __all__ failure in pycompat27: ImportError: cannot import name 'mswindows'
======================================================================
FAIL: Test that importing subprocess as a module works
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test_pycompat.py", line 11, in test_subprocess
    from kitchen.pycompat24.subprocess import Popen
ImportError: cannot import name 'mswindows'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test_pycompat.py", line 13, in test_subprocess
    tools.ok_(False, 'Unable to import pycompat24.subprocess as a module')
AssertionError: Unable to import pycompat24.subprocess as a module
======================================================================
FAIL: Test that the combining table we generate is equal to or a subset of what's in the current table
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/python3-python-kitchen-1.2.3-1.fc24/kitchen3/tests/test_text_display.py", line 46, in test_internal_generate_combining_table
    tools.assert_true(display._interval_bisearch(interval[0], old_table))
AssertionError: False is not true
----------------------------------------------------------------------
Ran 590 tests in 18.210s
FAILED (SKIP=3, errors=6, failures=4)
```

--- a/kitchen3/kitchen/pycompat27/subprocess/__init__.py
+++ b/kitchen3/kitchen/pycompat27/subprocess/__init__.py
@@ -43,7 +43,7 @@ warnings.warn('In python3, kitchen.pycompat27.subprocess is deprecated.'
 # All versions of python3 have a modern enough subprocess.  This module only
 # exists for backwards compatibility
 from subprocess import *
-from subprocess import list2cmdline, mswindows
+from subprocess import list2cmdline
 from subprocess import __all__
 
 # subprocess.MAXFD was removed in python-3.5
@@ -56,3 +56,11 @@ except ImportError:
         MAXFD = os.sysconf("SC_OPEN_MAX")
     except ValueError:
         MAXFD = 256
+
+# subprocess.mswindows was renamed in python-3.5
+# https://github.com/fedora-infra/kitchen/issues/12
+try:
+    from subprocess import mswindows
+except ImportError:
+    # Python 3.5
+    from subprocess import _mswindows as mswindows

